inventory/group_vars/all.yml

```diff
@@ -6,6 +6,7 @@ dns_adblock_lists: "{{ dns_adblock_lists_defaults + dns_adblock_lists_extras }}"
 dns_replace_address:
   - '([0]{1,3}\.){3}[0]{1,3} '  # Beware of the necessary trailing whitespace!
```

Comment:
Should we change this back to ([0-9]{1,3}\.){3}[0-9]{1,3}?  Changed this when adding the other task, so might be good to change this back to how it was originally

Reply:
Good thinking. I will add that and try it